src/common/utilities/GeneralHelper.ts

```diff
@@ -4,6 +4,7 @@ import * as strings from 'ControlStrings';
 
 export const IMG_SUPPORTED_EXTENSIONS = ".gif,.jpg,.jpeg,.bmp,.dib,.tif,.tiff,.ico,.png,.jxr,.svg";
 
+import * as _ from '@microsoft/sp-lodash-subset';
```

Comment:
Why do you need this `import`

Reply:
Hello @AJIXuMuK,

Removed this unwanted ```import``` in the latest commit

Thanks,
Nishkalank